easybuild/easyblocks/s/suitesparse.py

```diff
@@ -49,6 +50,14 @@
 class EB_SuiteSparse(ConfigureMake):
     """Support for building SuiteSparse."""
 
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """Define extra easyconfig parameters"""
+        extra_vars = {
+            'cmake_options': ['-DBLA_VENDOR=FlexiBLAS', "CMAKE_OPTIONS used by SuiteSparse since v6.0", CUSTOM],
```

Comment:
The code should check which BLAS option it can use, i.e. if flexiblas is available else if MKL is available or if openblas etc. Never hardcode things that can be checked for.

Reply:
And probably also make sure not to override a setting coming from the easyconfig, see the cmakemake easyblock for how to do that.